chrome/script.browserify.js

```diff
@@ -69,7 +70,7 @@ function view() {
     ]),
 
     // results
-    m("div.results", results)
+    m("div.results", { onkeydown: keyHandler }, results)
```

Comment:
As you can see handler is assigned to a container of results and container of input. I did it because I didn't want to change structure without asking you first. Alternatively, we can add a new extra container to render or use native API to register the handler.

Reply:
I don't mind wrapping both containers in a some kind of parent container, and assigning the onkeydown to it. But if for whatever reason it gets complicated, the current solution is also fine.